src/components/MessageList/VirtualizedMessageList.tsx

```diff
@@ -206,6 +206,7 @@ const VirtualizedMessageListWithContext = <
         if (style) acc[message.id] = style;
         return acc;
       }, {} as Record<string, GroupStyle>),
```

Comment:
```suggestion
      }, {}),
```
Not part of the PR so feel free to ignore this comment but noticed this casting which could be refactored to this:
```ts
array.reduce<Record<string, GroupStyle>>(...)
```

Reply:
@arnautov-anton added generic type def to all reduce calls